crates/net/downloaders/src/bodies/concurrent.rs

```diff
@@ -61,7 +61,6 @@ where
             stream::iter(headers.into_iter().map(|header| {
                 (|| self.fetch_body(header))
                     .retry(ExponentialBackoff::default().with_max_times(self.retries))
-                    .when(|err| err.is_retryable())
```

Comment:
does this mean that now we retry on all errors?

Reply:
correct, the timeouts are handled by the network, so only validation and internal errors are left

Reply:
:+1: